project-base/storefront/components/Pages/Order/PaymentConfirmation/PaymentInProcess.tsx

```diff
@@ -23,17 +22,14 @@ export const PaymentInProcess: FC<PaymentInProcessProps> = ({ orderUrlHash }) =>
     useGtmPageViewEvent(gtmStaticPageViewEvent);
 
     return (
-        <Webline className="mt-8 lg:mt-10 vl:mt-20">
-            <div className="flex items-center gap-6">
-                <HeartIcon className="text-green h-20 w-20" />
-                <h1 className="mb-0">
-                    {t('The payment is being processed, you can check the status on the order detail page')}
-                </h1>
-            </div>
-            <a className="w-fit" href={orderDetailUrl}>
-                {t('Show order detail')}
-                <ArrowIcon className="h-3 w-3 -rotate-90" />
-            </a>
-        </Webline>
+        <ConfirmationPageContent
+            content={t('You can check the status on the order detail page.')}
+            heading={t('The payment is being processed')}
+            AdditionalContent={
```

Comment:
please, use camelCase for `AdditionalContent`

Reply:
Refactored this so that instead of `AdditionalContent` children will be used for extra content